Dockerfile

```diff
@@ -1,10 +1,13 @@
 FROM python:3.8-slim-buster
-RUN apt-get update
+
+# Change working directory.
 WORKDIR /app
+# Update PIP.
+RUN pip install --upgrade pip
+# Install requirements from file.
 COPY requirements.txt requirements.txt
 RUN pip install -r requirements.txt
-COPY . .
```

Comment:
Whole directory contains not needed for the docker stuff like readmes and wget.exe that not needed inside container. Just skipping all of them will decrease container size.